src/Microsoft.AspNetCore.Blazor.Browser/Interop/ArgumentList.cs

```diff
@@ -1,260 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Concurrent;
-using System.Linq;
-using System.Reflection;
-
-namespace Microsoft.AspNetCore.Blazor.Browser.Interop
-{
-    internal class ArgumentList
```

Comment:
RIP

Reply:
Yep, the updated equivalent to this is: https://github.com/aspnet/Blazor/pull/1033/files#diff-b80ce524aaadc1a948d2f5b50db36160R77

Instead of having to create a special wire format for args arrays and then a lot of .NET-side logic for parsing and extracting values, it was possible just to `JSON.serialize` the whole `args` array on the JS side with no preprocessing, and then do a two-phase deserialize on the .NET side. It shouldn't cost anything extra in perf as in effect it was doing this before but just not retaining the information it needed.